examples/cc_gwlb/terraform.tfvars

```diff
@@ -105,50 +99,56 @@
 #cc_subnets                                 = ["10.x.y.z/24","10.x.y.z/24"]
 #route53_subnets                            = ["10.x.y.z/24","10.x.y.z/24"]
 
-
 ## 11. Number of Workload VMs to be provisioned in the workload subnet. Only limitation is available IP space
 ##    in subnet configuration. Only applicable for "base" deployment types. Default workload subnet is /24 so 250 max
 
 #workload_count                             = 2
 
-
 ## 12. Tag attribute "Owner" assigned to all resoure creation. (Default: "zscc-admin")
 
 #owner_tag                                  = "username@company.com"
 
-
 ## 13. By default, Cloud Connectors are configured with a callhome IAM policy enabled. This is recommended for production deployments
 ##     The policy creation itself does not provide any authentication/authorization access. IAM details are still required to be provided
 ##     to Zscaler in order to establish a trust relationship. Uncomment if you do not want this policy created. (Default: true)
 
 #cc_callhome_enabled                        = false
 
-
 ## 14. By default, GWLB deployments are configured as zonal. Uncomment if you want to enable cross-zone load balancing
 ##     functionality. Only applicable for gwlb deployment types. (Default: false)
 
 #cross_zone_lb_enabled                      = true
 
+## 15. Gateway loadbalancing hashing algorith. Zscaler recommended default is 2-tuple (source_ip_dest_ip).
```

Comment:
algorith->agorithm

Reply:
fixed in commit 13cda2542b0e1ecb28a8e65b6f59cd5309b13dd5